### ISSUE TITLE
Tools not loaded - Eaerly bound Gen V2

### ISSUE THREAD

**Issue by GWawrzeniecki** [daryllabar/DLaB.Xrm.XrmToolBoxTools#527]
2 latest versions

![image](https://github.com/user-attachments/assets/52122c46-2a1a-4202-a219-d248dc22e77f)

**Comment by daryllabar** [comment]
Sorry, just saw this, did you get it resolved?

**Comment by GWawrzeniecki** [comment]
yes, I got copy of XrmToolbox folder from a colleague and downloaded older version of the generator
